Cargo.toml

```diff
@@ -32,11 +32,11 @@ prost           = { version = "0.12.3", default-features = false, features = ["p
 bytes           = { version = "1.2",  default-features = false }
 tonic           = { version = "0.10",  default-features = false, optional = true }
 serde           = { version = "1.0",  default-features = false, optional = true }
-schemars        = { version = "0.8",  optional = true }
```

Comment:
Why the `jsonshcema` attributes have been discarded?

Reply:
Ah! This was in error. Mistakenly bundled this up with serde attr removals. It doesn't help that schemars and serde kinda look similar 😅 . Restored.